src/app/constants/editioncard.js

```diff
@@ -0,0 +1,5 @@
+// Constants
+export const MAX_TITLE_LENGTH = 80;
+export const MAX_SUBTITILE_LENGTH = 80;
```

Comment:
Is this coming from the design system? I would think that we could allow the subtitle to be longer than the main title.

Reply:
No, we can make it whatever we like, I just picked a number at random haha

Reply:
Ok, cool. Can we make it something like 120? Just thinking that generally the structure (especially for academic books): "Short Title: A Longer Descriptive Title, with stuff" and it would be good to get more of that descriptive sub title in the design if possible

Reply:
Yeah!  I have yet to find a book with a subtitle... but that makes lots of sense!